ztp/siteconfig-generator/siteConfig/siteConfigBuilder.go

```diff
@@ -276,6 +298,39 @@ func translateTemplateKey(key string) (string, error) {
 	return "", fmt.Errorf("Key %q could not be translated", key)
 }
 
+func (scbuilder *SiteConfigBuilder) buildBiosConfigMapCR(clusterId int, siteConfigTemp SiteConfig, nodeId int) (interface{}, string, error) {
```

Comment:
The config map generated from the extra-manifests is consumed by the assisted-service to populate the MachineConfig for the spoke clusters.  The data is mostly passed through. It would be difficult for the assisted-service to search for BIOS configuration in the same config map.